x-pack/auditbeat/module/system/host/host.go

```diff
@@ -15,7 +15,7 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/cespare/xxhash"
+	xxhash "github.com/cespare/xxhash/v2"
```

Comment:
is the "rename" of `v2` to `xxhash` required? Is it added by automation? If a package only imports `vX` I think one can ommit the renaming part.

Reply:
No. For some reason `goimports` kept removing the line until I added the alias. But now I am able to remove it. So I assume it was some kind of  PEBKAC. So I removed thos now.